### PR TITLE
qa_crowbarsetup: Remove duplicated manila code

### PR DESCRIPTION
**Pull request by toabctl**
Detecting the uuid and floating IP of the VM is now handled
in a single place.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3149,12 +3149,25 @@ function get_ceph_nodes()
...
+function manila_service_instance_get_uuid()
+{
+    local vm_uuid=`openstack --os-project-name manila-service server show manila-service -f value -c id`
+    test -n "$vm_uuid" || complain 91 "uuid from manila-service instance not available"
+    echo $vm_uuid
+}
+
+function manila_service_instance_get_floating_ip()
+{
+    local vm_uuid=`manila_service_instance_get_uuid`
```
Comment (abel-navarro) [inline scripts/qa_crowbarsetup.sh]: careful with this way of calling a function. I'm not sure if it's correct.

Comment (toabctl) [inline scripts/qa_crowbarsetup.sh]: it works. See other examples in the script or i.e. http://www.linuxjournal.com/content/return-values-bash-functions

**Comment by toabctl** [comment]
Worked with ```develcloud7``` (at least the critial point passed). Now retriggering with ```develcloud6```

**Comment by abel-navarro** [comment]
+1

**Comment by aplanas** [comment]
+1

**Pull request merged**